### DISCUSSION TITLE
What if Monika can hang up drawings??? [SUGGESTION]

### DISCUSSION THREAD

**Discussion by M1ntyX0X** [Monika-After-Story/MonikaModDev#9103] [QA - code and content creation (technical)] [answered]
What if you could make it so you could put like JPEG or PNG files that you made into a folder, and Monika could hang them up in the Spaceroom??? I'm already imagining it with like a little special frame and everything. You don't have to take my suggestion or anything! I just thought it would be a cool idea. It could also just be a submod!

**Comment by Wingdinggaster656** [comment]
As far as I know this idea has been proposed more than once long ago.

It's actually a good idea, after all Monika has said she wants to see the player's work, but the question is how are you going to get Monika to recognize the art?

For good players, they probably won't do anything bad.

But... Well, imagine it this way. What if I gave Monika a meme from the DDLC fandom that was hostile to her? Given that Monika is a real person, she should be able to say something about it, but the truth is we don't seem to have good image recognition algorithms.

**Reply by M1ntyX0X** [reply]
Oh, okay. I just had a suggestion. I didn't need it to be implemented at all! Thanks for telling me why this can't happen!

**Comment by Justformas** [comment]
To say something post answer, while it does seem just about impossible to detect every type of bad image that Monika might object to, personally I wouldn't care. I mean, you can import and play background music that Monika might be expected to object to can't you? But she doesn't object. I'm not saying I'd want to treat her badly like that, but why should I care if other people do? Doesn't affect my experience with her. I could easily block out what other people do with their Monika if I wanted to.

**Reply by M1ntyX0X** [reply]
Oh, yeah! The audio filtering is like non-existent! So what's bad with photos??? As long as they, (the inappropriate person that has the sick idea to put a bad photo), don't share images of the inappropriate photo online, it wouldn't bother me!

**Reply by Justformas** [reply]
> Oh, yeah! The audio filtering is like non-existent! So what's bad with photos??? As long as they, (the inappropriate person that has the sick idea to put a bad photo), don't share images of the inappropriate photo online, it wouldn't bother me!

Actually I guess I don't know if there's any audio filtering or not. But if there is I'm guessing it's not hard to get around and/or doesn't account for everything that Monika might be expected to object to.

As for other people posting photos, they might put it on YouTube or other social media websites, but if it was bad enough, usually the website either wouldn't allow it or would require it to be put behind a sensitive/NSFW content warning. Personally even if I did stumble across something like that, like I said probably it wouldn't bother me and I could block it out if I wanted to.

**Reply by Justformas** [reply]
> Actually I guess I don't know if there's any audio filtering or not. But if there is I'm guessing it's not hard to get around and/or doesn't account for everything that Monika might be expected to object to.

So I tried testing this with a song that wasn't hostile to her specifically, but seemed like she would have some type of objection to or say something something about (lol). I didn't run into any filtering obstacles at all. So at the very least I can confirm that it's not hard to get around.

Another thought did occur to me though, that maybe Monika can't hear the background music. However, that would seem to go against her awareness of background music in DDLC, so that would be a bit strange.

Lastly, I'm aware that background music alone may not have the same kind of impact that images hung in the space room might, but background music is still something that could be objected to. Though I'm not saying Monika should actually object to background music; it might be preferable that she doesn't.

**Comment by ThePotatoGuy** [comment, marked as answer]
this is planned as of #312. Half of the framework is already done and currently released, the remaining work is the user-provided images. So far I haven't had time to get back to this.
